### DISCUSSION TITLE
Newbie Question

### DISCUSSION THREAD

**Discussion by roblind0913** [emrahcom/galaxy#1] [Q&A] [answered]
Forgive me if I am asking stupid questions, I am new and figuring things out as I go.  

I am trying to setup galaxy on a Debian 11 Digital Ocean server.  I keep getting the same error and I am not sure what I am doing wrong.  

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: galaxy-ui@0.0.1
npm ERR! Found: vite@3.2.5
npm ERR! node_modules/vite
npm ERR!   dev vite@"^3.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^4.0.0" from @sveltejs/kit@1.0.0-next.589
npm ERR! node_modules/@sveltejs/kit
npm ERR!   dev @sveltejs/kit@"next" from the root project
npm ERR!   peer @sveltejs/kit@"^1.0.0-next.587" from @sveltejs/adapter-auto@1.0.0-next.91
npm ERR!   node_modules/@sveltejs/adapter-auto
npm ERR!     dev @sveltejs/adapter-auto@"next" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

I tried the force and legacy options without any success.

I would appreciate any assistance.

Thanks.

**Comment by bayraktarulku** [comment]
dependency on vite version ^4.0.0, but you already have vite version 3.2.5 installed in your project. 
- Check if there is a newer version of vite available that is compatible with the version. You can do this by running the command npm show vite version. If there is a newer version available, you can update vite by running npm install vite@[new_version]
- If there is no newer version of vite available, or if updating vite does not resolve the issue, you can try installing the galaxy package with the --legacy-peer-deps. This will allow the installation to proceed even if there are conflicts in the peer dependencies. However, that using this may result in an incorrect or broken dependency.

**Comment by emrahcom** [comment]
There may be an error on my side. I will check it today

**Comment by emrahcom** [comment]
It should be fixed now. Please download the installer config (`eb-galaxy.conf`) again before starting the installation

```bash
wget -O eb-galaxy.conf https://raw.githubusercontent.com/emrahcom/galaxy/main/installer/eb-galaxy.conf
```

**Comment by roblind0913** [comment, marked as answer]
Thanks I will give it a try.  On Dec 25, 2022, at 11:41 AM, emrah ***@***.***> wrote:﻿
It should be fixed now. Please download the installer config (eb-galaxy.conf) again before starting the installation
wget -O eb-galaxy.conf https://raw.githubusercontent.com/emrahcom/galaxy/main/installer/eb-galaxy.conf

**Reply by roblind0913** [reply]
Thanks, that worked.

**Comment by qiujunhong1** [comment]
大哥们我也是新手，我部署后主页可以访问登录和注册报404啥情况呀，我就局域网部署

**Comment by emrahcom** [comment]
Hi @qiujunhong1, are there any errors in the browser console logs?

**Reply by qiujunhong1** [reply]
app.DphrDOte.js:2 
 Dt: Not found: /self-service/registration/browser
    at J (https://192.168.110.52/_app/immutable/chunks/CFjML2r5.js:1:22195)
    at async Module.Gn (https://192.168.110.52/_app/immutable/chunks/CFjML2r5.js:1:13183)
handleError	@	app.DphrDOte.js:2
G	@	CFjML2r5.js:1
J	@	CFjML2r5.js:1
await in J		
Gn	@	CFjML2r5.js:1
await in Gn		
(匿名)	@	browser:36
Promise.then		
(匿名)	@	browser:35

**Reply by qiujunhong1** [reply]
我是打算在局域网部署，我没有两个域名，我就用的ip加端口号的形式部署的export GALAXY_FQDN="192.168.110.52"
export KRATOS_FQDN="192.168.110.52"
export SKIP_DNS_CHECK=true  # 跳过域名验证

# 端口配置（默认端口，避免冲突）
export APP_UI_PORT=8080
export KRATOS_PUBLIC_PORT=4433
export KRATOS_ADMIN_PORT=4434
export APP_API_PORT=3000
但是我部署好了后https://192.168.110.52:8080/无法访问，https://192.168.110.52/可以访主页
![1](https://github.com/user-attachments/assets/5457925a-837f-4b65-b65d-eda75fe6d574)
![2](https://github.com/user-attachments/assets/4493a7ed-2178-40c3-b036-f2d4ddf4488e)

**Reply by qiujunhong1** [reply]
![Uploading 3.png…]()

**Comment by emrahcom** [comment]
You should use separate domain names for `GALAXY_FQDN` and `KRATOS_FQDN`.
Don't use IP addresses.

**Reply by qiujunhong1** [reply]
So this project must have two subdomains to be deployed?

**Reply by emrahcom** [reply]
Correct.

For example I use `app.eparto.net` and `id.eparto.net` for [Eparto](https://eparto.net)
